sklearn/tree/tests/test_export.py

```diff
@@ -63,6 +63,25 @@ def test_graphviz_toy():
 
     assert contents1 == contents2
 
+    # Test with feature_names (escaped)
+    contents1 = export_graphviz(clf,
+                                feature_names=["feature\"0\"", "feature\"1\""],
+                                out_file=None)
+    contents2 = 'digraph Tree {\n' \
+                'node [shape=box] ;\n' \
+                '0 [label="feature\\"0\\" <= 0.0\\n' \
+                'gini = 0.5\\nsamples = 6\\n' \
+                'value = [3, 3]"] ;\n' \
+                '1 [label="gini = 0.0\\nsamples = 3\\nvalue = [3, 0]"] ;\n' \
+                '0 -> 1 [labeldistance=2.5, labelangle=45, ' \
+                'headlabel="True"] ;\n' \
+                '2 [label="gini = 0.0\\nsamples = 3\\nvalue = [0, 3]"] ;\n' \
+                '0 -> 2 [labeldistance=2.5, labelangle=-45, ' \
+                'headlabel="False"] ;\n' \
+                '}'
```

Comment:
To improve readability may I suggest the following idiom:

```python
from textwrap import dedent

...
    contents1 = export_graphviz(clf,
                                feature_names=["feature\"0\"", "feature\"1\""],
                                out_file=None)
    contents2 = dedent("""\
        digraph Tree {
        node [shape=box] ;
        ...
        }""")


```

Reply:
I was hesitant to use a convention different from the surrounding code, since all test cases in the file use this form. Do you think it's worth changing only for this new case?

Reply:
@ogrisel So would you prefer the code using the same convention as the surrounding test cases? Or the one you suggested?